rest/taskrouter/twiml/example1/example/example.5.x.cs

```diff
@@ -0,0 +1,22 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using System;
+using System.Net;
+using Twilio.TwiML;
+
+class Example
+{
+    public static HttpListenerResponse SendResponse(HttpListenerContext ctx)
+    {
+        HttpListenerRequest request = ctx.Request;
+        HttpListenerResponse response = ctx.Response;
+
+        response.StatusCode = (int)HttpStatusCode.OK;
+        response.ContentType = "application/xml";
```

Comment:
Most of the time we want to use "text/xml" as the default content type. Please take a look at https://www.ietf.org/rfc/rfc2376.txt.